masterlist.yaml

```diff
@@ -19666,6 +19666,10 @@ plugins:
     inc:
       - 'Blues Skyrim.esp'
       - 'ETaC - Complete.esp'
+  - name: 'JKs_Skyrim_No_Lights_Patch.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/6289/' ]
+    after:
+      - 'JKs Skyrim - Rob''s Bug Fixes.esp'
```

Comment:
Looking at the two in xEdit, they don't actually conflict in any major way. Just the MHDT subrecord of the Tamriel worldspace which probably isn't important to either plugin (also which will most likely get overwritten by something else anyway & I'm not sure even does anything) & a water subrecord in __00009349__ that's probably unintentionally reverted by `JKs_Skyrim_No_Lights_Patch.esp` when it loads after.

That being said, I can see it being the case that either plugin may be updated in a way where load order would matter. In that case, you can use the following regex plugin to cover what appears to be most patches. Feel free to add rules for any others that were missed as you see fit. I've also added a location name & converted the single-element file map to flow style. You'll have to manually place it above all of the other patches but just below `JKs Skyrim.esp`:

```yaml
  - name: 'JKs[ _]Skyrim_.*_Patch\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/6289/'
        name: 'JK''s Skyrim'
    after: [ 'JKs Skyrim - Rob''s Bug Fixes.esp' ]
```